launch/app/pom.xml

```
@@ -55,7 +55,14 @@
       <version>${project.version}</version>
       <scope>runtime</scope>
     </dependency>
-    
+
+    <dependency>
+      <groupId>org.openhab.addons.bundles</groupId>
+      <artifactId>org.openhab.binding.zigbee</artifactId>
```

The BOM added in https://github.com/openhab/org.openhab.binding.zigbee/pull/453 has a different artifactId:

```suggestion
      <artifactId>org.openhab.binding.zigbee.bom</artifactId>
```

---

Thanks, I have force pushed the correction.